### PR TITLE
pokeget 1.4.3 (new formula)

### PR DESCRIPTION
**Pull request by talwat**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/pokeget.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Github Link: https://github.com/talwat/pokeget
```
Comment (carlocab) [inline Formula/pokeget.rb:1]:
```suggestion
```

File: Formula/pokeget.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+class Pokeget < Formula
+  desc "Bash script that displays cool sprites of pokemon in your terminal"
+  homepage "https://github.com/talwat/pokeget"
+  url "https://github.com/talwat/pokeget/releases/download/1.4.3/pokeget_1.4.3-src.tar"
+  sha256 "bcd94f2557856b47be7c338eddfdcc7a832f21545312a0432bb847a18b14b486"
+  license "MIT"
+  head "https://github.com/talwat/pokeget.git", branch: "main"
+
+  depends_on "make" => :build
```
Comment (carlocab) [inline Formula/pokeget.rb:10]:
```suggestion
```
You don't need this.

File: Formula/pokeget.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  on_macos do
+    depends_on "coreutils"
+  end
+
+  def install
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    system "#{bin}/pokeget", "version"
```
Comment (carlocab) [inline Formula/pokeget.rb:20]: Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

**Review by carlocab — COMMENTED** [review]
Thanks for the PR. This looks like a nice piece of software but it doesn't meet the notability requirements for inclusion into Homebrew/core.

See the CI log, or run `brew audit --new pokeget` in your PR branch for details.

If you wish, you can host this formula in your own tap. Here are some docs to help you get started:

https://docs.brew.sh/Taps
https://docs.brew.sh/Interesting-Taps-and-Forks
https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap
https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/

Please address my comments below if you choose to submit this again in the future.

**Comment by talwat** [comment]
Should I reopen this when I have fixed the issues, or should I make a new PR?

**Comment by carlocab** [comment]
You can open a new PR.

**Pull request closed without merging**